src/libasr/pass/pass_manager.h

```diff
@@ -74,6 +75,7 @@ namespace LCompilers {
             {"select_case", &LFortran::pass_replace_select_case},
             {"loop_vectorise", &LFortran::pass_loop_vectorise},
             {"array_dim_intrinsics_update", &LFortran::pass_update_array_dim_intrinsic_calls},
+            {"pass_list_concat", &LFortran::pass_list_concat},
```

Comment:
Why does it generate an error?
```
error: could not convert ‘{{"do_loops", LFortran::pass_replace_do_loops}, {"global_stmts", LFortran::pass_wrap_global_stmts_into_function}, {"implied_do_loops", LFortran::pass_replace_implied_do_loops}, {"array_op", LFortran::pass_replace_array_op}, {"arr_slice", LFortran::pass_replace_arr_slice}, {"print_arr", LFortran::pass_replace_print_arr}, {"print_list", LFortran::pass_replace_print_list}, {"class_constructor", LFortran::pass_replace_class_constructor}, {"unused_functions", LFortran::pass_unused_functions}, {"flip_sign", LFortran::pass_replace_flip_sign}, {"div_to_mul", LFortran::pass_replace_div_to_mul}, {"fma", LFortran::pass_replace_fma}, {"sign_from_value", LFortran::pass_replace_sign_from_value}, {"inline_function_calls", LFortran::pass_inline_function_calls}, {"loop_unroll", LFortran::pass_loop_unroll}, {"dead_code_removal", LFortran::pass_dead_code_removal}, {"forall", LFortran::pass_replace_forall}, {"select_case", LFortran::pass_replace_select_case}, {"loop_vectorise", LFortran::pass_loop_vectorise}, {"array_dim_intrinsics_update", LFortran::pass_update_array_dim_intrinsic_calls}, {"pass_list_concat", LFortran::pass_list_concat}, {"pass_array_by_data", LFortran::pass_array_by_data}}’ from ‘<brace-enclosed initializer list>’ to ‘std::map<std::__cxx11::basic_string<char>, void (*)(Allocator&, LFortran::ASR::TranslationUnit_t&, const LCompilers::PassOptions&)>’
   80 |         };
      |         ^
      |         |
      |         <brace-enclosed initializer list>
```

Reply:
https://github.com/lcompilers/lpython/pull/1119/files#r983134603